ldp/alg/runners.py

```diff
@@ -298,7 +298,7 @@ class OfflineTrainerConfig(BaseModel):
         ge=1,
     )
     clear_ctx_at_each_iter: bool = False
-    # TODO: add some concept of eval loops
+    num_epochs: int = 1
```

Comment:
```suggestion
    num_epochs: int = Field(default=1, ge=0)
```

Just adding a nice validator for negative values